multimodal/src/autogluon/multimodal/data/labelencoder_ner.py

```diff
@@ -35,6 +36,7 @@ def fit(self, y: pd.Series, x: pd.Series):
         _, entity_groups = self.extract_ner_annotations(y)
         self.unique_entity_groups = self.ner_special_tags + entity_groups
         self.entity_map = {entity: index for index, entity in enumerate(self.unique_entity_groups)}
+        self.config.entity_map = self.entity_map
```

Comment:
Do we need to put entity_map under `self.config`? Or we just need to keep `self.entity_map`?

Reply:
need this entity map in dataprocessor, that's why it is put in config. special tags such as "O" is also in config.

Reply:
Shall we extend the`NerProcessor` to include the `config` keyword then?

Reply:
```python
class NerProcessor:
    """
    Prepare NER data for the model specified by "prefix".
    """
    def __init__(
        self,
        model: nn.Module,
        max_len: Optional[int] = None,
        entity_map: Optional[dict] = None,
        config: Optional[DictConfig] = None,
    ):
```

Reply:
already add self.config in nerprocessor. entity_map can be accessed by self.config.entity_map.